CMakeLists.txt

```diff
@@ -205,6 +207,12 @@ endif()
 # Vulkan
 if(LOVR_USE_VULKAN)
   list(APPEND LOVR_INCLUDES deps/vulkan-headers/include)
+
+  if(LOVR_LINK_VULKAN)
+    pkg_search_module(VULKAN REQUIRED vulkan)
```

Comment:
Wouldn't `FindVulkan` be a little more canonical/robust?

Reply:
Huh... TIL, what a random feature to include in a build system. Thanks